pkg/app/app.go

```diff
@@ -131,7 +131,27 @@ func CreateApp() App {
 	return app
 }
 
+// TODO: should this be a go routine?
+// if so, what happens if the http listener is started and getting requests
+// before we have recovered our state?
+func (a *App) Recover() {
+	msg, err := a.broker.Recover()
+
+	if err != nil {
+		a.log.Error(err.Error())
+	}
+
+	a.log.Notice(msg)
+}
+
 func (a *App) Start() {
+	a.log.Info("Initiating Recovery Process")
+	// TODO: probably return an error or some sort of message such that we can
+	// see if we need to go any further. Do I even need a method on the App?
+	// Should this just be called from main before we call Start? I feel like
+	// main shouldn't give a crap if we need to recover or not.
+	a.Recover()
```

Comment:
One thing that's not ideal is we run recovery every time without choice.  I'm worried that if a recovery breaks on something a user will never be able to get past here.